utils/display.go

```diff
@@ -190,6 +190,46 @@ func TransformsTable(transforms []*meroxa.Transform, hideHeaders bool) string {
 	return ""
 }
 
+func ConnectorTable(connector *meroxa.Connector) string {
+	mainTable := simpletable.New()
+	mainTable.Body.Cells = [][]*simpletable.Cell{
+		{
+			{Align: simpletable.AlignRight, Text: "ID:"},
+			{Text: fmt.Sprintf("%d", connector.ID)},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Name:"},
+			{Text: connector.Name},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Type:"},
+			{Text: connector.Type},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Streams:"},
+			{Text: formatStreams(connector.Streams)},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "State:"},
+			{Text: connector.State},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Pipeline:"},
+			{Text: strings.Title(connector.PipelineName)},
```

Comment:
I would personally not capitalize the name of a pipeline to avoid cases in which we copy that string elsewhere and for some reason we are case sensitive.

```suggestion
			{Text: connector.PipelineName},
```